test/test_table_view.cpp

```diff
@@ -1174,13 +1175,13 @@ TEST(TableView_LowLevelSubtables)
 TEST(TableView_HighLevelSubtables)
 {
     Table t;
-    t.add_column(type_Int, "val");
-    DescriptorRef sub;
-    t.add_column(type_Table, "subtab", &sub);
-    sub->add_column(type_Int, "val");
-    DescriptorRef subsub;
-    sub->add_column(type_Table, "subtab", &subsub);
-    subsub->add_column(type_Int, "value");
+    t.add_column(type_Int, "val");
```

Comment:
Do you know what changed here? Git diff is not showing anything useful. Line endings?